apps/prairielearn/src/question-servers/freeform.js

```diff
@@ -1984,7 +1984,7 @@ module.exports = {
     try {
       const commitHash = await courseUtil.getOrUpdateCourseCommitHashAsync(course);
       const dataHash = objectHash({ data, context }, { algorithm: 'sha1', encoding: 'base64' });
-      return `${commitHash}-${dataHash}`;
+      return `question:${commitHash}-${dataHash}`;
```

Comment:
Adding a scope to cache keys is generally a good idea, especially now that it's being used for multiple things.